CHANGES.rst

```diff
@@ -1,6 +1,8 @@
 0.9.3dev
 --------
-
+- Fixed a bug that was introduced when the binning was switched to the PypeIt convention.
+- Fixed a bug whereby 2d images were not being saved if no objects were detected.
```

Comment:
Are you sure this was a `bug`?

Reply:
Yes it was a bug since those images were meant to be saved all along, and also the code uses the existence of 2d images to decide whether a frame has been reduced.